faraday/simpleconfig.py

```diff
@@ -143,6 +143,7 @@
 bootbitmask.reverse()
 logger.info("UNITCONFIGURED: {0}".format(bootbitmask[0]))
 logger.info("REDLEDTX: {0}".format(bootbitmask[1]))
+logger.info("GREENLEDRX: {0}".format(bootbitmask[5]))
```

Comment:
Correct, I was confused by `bootbitmask.reverse()`. Thanks for clarifying, edit made.